app/account/edit/components/account-form.tsx

```diff
@@ -0,0 +1,155 @@
+import { useState } from 'react'
+import { useForm, Controller } from 'react-hook-form'
+import {
+  Box,
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+  Image,
+  VStack,
+  HStack,
+  Flex,
+  Input
+} from '@chakra-ui/react'
+
+import { MdAccountCircle } from 'react-icons/md'
+import { AccountSchema, accountResolver } from '@/account/schema'
+import { PrimaryButton, SecondaryButton } from '@/components/button'
+import { InputForm } from '@/components/input'
+import { Loading } from '@/components/loading'
+import { useUserId } from '@/providers/session-provider'
+import { useUploadFile, useUserUpdate } from '../../hooks'
+
+export const AccountEditForm = ({
+  name,
+  email,
+  profile_picture_url
+}: AccountSchema) => {
+  const userId = useUserId()
+  const { uploadFile } = useUploadFile()
+  const { updateUser, isLoading } = useUserUpdate()
+
+  const [selectedImage, setSelectedImage] = useState<File | null>(null)
+
+  const {
+    register,
+    handleSubmit,
+    control,
+    formState: { errors }
+  } = useForm<AccountSchema>({
+    defaultValues: {
+      name: name,
+      email: email,
+      profile_picture_url: profile_picture_url,
+      uploaded_image_file: null
+    },
+    resolver: accountResolver
+  })
+
+  const createHandler = handleSubmit(async (data: AccountSchema) => {
+    if (selectedImage) {
+      await uploadFile(profile_picture_url, selectedImage, userId)
+    }
+
+    updateUser(data.name, data.email)
+  })
```

Comment:
After updating the user information, you show the form for a second and then redirect to the user details. Could you fix it? We don't want to show the account edit page for a second before redirecting.

Reply:
I've worked on it a little. 
Here is how it looks 

Uploading Screen Recording 2024-02-27 at 11.14.04.mov…